docs/Getting Started/Arch Linux/Arch Linux Root on ZFS.rst

```diff
@@ -127,24 +127,18 @@ Prepare the Live Environment
 
 #. Import keys of archzfs repository::
 
-    curl -O https://archzfs.com/archzfs.gpg
-    pacman-key -a archzfs.gpg
-    pacman-key --lsign-key DDF7DB817396A49B2A2723F7403BD972F75D9D76
```

Comment:
Personally, I think it's better to keep the key ID hardcoded here. I suppose if someone's typing it in, it's harder, but if they're copying-and-pasting, it's the same. And it's not clear to me who controls git.io/JtHEx. And same for the couple places below.

Reply:
Both key-id link and mirrorlist link is maintained by myself, at [this repo](https://github.com/ne9z/mirrorlist-archzfs).

Using a link instead of a hard-coded value is not only for convenience of the user, but also for maintaining the guide.

This key-id [has been changed before](https://github.com/archzfs/archzfs/issues/262). It might be easy for me to replace 5 occurrences in these guides, but not so much for replacing its derivatives. If someone copy-and-pasted the code to their own blog, script or something, it's up to them to update the key-id. Just a quick search on [Google](https://www.google.com/search?q=%22DDF7DB817396A49B2A2723F7403BD972F75D9D76%22) turned up many instances.

Reply:
For clarification:
- git.io/JtHEx points to https://github.com/ne9z/mirrorlist-archzfs/raw/master/key-id
- git.io/JtHEh points to https://github.com/ne9z/mirrorlist-archzfs/raw/master/mirrorlist-archzfs